server/api/index.js

```diff
@@ -0,0 +1,10 @@
+import hello from './hello'
```

Comment:
o ya prolly just kill hello while you're at it, since we have something that works here